core/tests/protractor_desktop/wipeout.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for user preferences.
+ */
+var DeleteAccountPage = require('../protractor_utils/DeleteAccountPage.js');
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
```

Comment:
```suggestion
var ExplorationEditorPage = require(
  '../protractor_utils/ExplorationEditorPage.js');
```
Nit: Break after parenthesis

Reply:
Done.